src/content_sao.h

```diff
@@ -121,6 +121,7 @@ class LuaEntitySAO : public UnitSAO
 	v3f getAcceleration();
 
 	void setTextureMod(const std::string &mod);
+	std::string getTextureMod();
```

Comment:
I don't think this code is in any way time critical nor is the result a huge amount of text.
So unless there's a new rule in coding guidelines demanding this I'd prefere not to change this.
I agree that the copy could be avoided but imho using parameters as return value is way less readable then using real return values.